sqs/src/test/scala/akka/stream/alpakka/sqs/scaladsl/SqsSpec.scala

```diff
@@ -188,6 +188,29 @@ class SqsSpec extends FlatSpec with Matchers with DefaultTestContext {
     probe.cancel()
   }
 
+  it should "pull messages from a fifo queue following the same production order" taggedAs Integration in {
+    val queue = randomFifoQueueUrl()
+    implicit val awsSqsClient = sqsClient
+
+    val messages = for (i <- 0 until 10) yield s"Message - $i"
+
+    messages
+      .map(msg => {
```

Comment:
Isn't this `foreach`? or just adding it to the `for` expression?

Reply:
Yes, indeed. Thanks for the observation. Suggestion applied!